patches/chrome-tools-build-mac-infoplist_strings_util.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/tools/build/mac/infoplist_strings_util.cc b/chrome/tools/build/mac/infoplist_strings_util.cc
+index 89801229291a63e69574bc13cfe563aac360012f..038a9a66222dedad219c83b06be3e53a00b70f94 100644
+--- a/chrome/tools/build/mac/infoplist_strings_util.cc
++++ b/chrome/tools/build/mac/infoplist_strings_util.cc
+@@ -153,6 +153,7 @@ int main(int argc, char* const argv[]) {
+     // Fetch the strings.
+     std::string name = LoadStringFromDataPack(branded_data_pack.get(), cur_lang,
+                                               name_id, name_id_str);
++    name = BRAVE_PRODUCT_STRING;
```

Comment:
it seems like we shouldn't have to do this at all. The branded data pak should already have the correct value, no?

Reply:
@simonhong might have a better idea if this is still needed. His original explanation for the patch: https://github.com/brave/brave-core/pull/298#issuecomment-409420123